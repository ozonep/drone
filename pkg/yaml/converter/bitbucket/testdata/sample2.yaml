pipelines:
  branches:
    feature/*:
      - step:
          name: Test
          image: node:latest
          script:
            - npm install
            - npm test
  default:
    - step:
        name: Build and test
        image: node:8.5.0
        caches:
          - node
        script:
          - npm install
          - npm test
          - npm build
        artifacts:
          - dist/**
    - step:
        name: Integration test
        image: node:8.5.0
        caches:
          - node
        services:
          - postgres
        script:
          - npm run integration-test
    - step:
        name: Deploy to beanstalk
        image: python:3.5.1
        script:
          - python deploy-to-beanstalk.py
 
definitions:
  services:
    postgres:
      image: postgres:9.6.4
