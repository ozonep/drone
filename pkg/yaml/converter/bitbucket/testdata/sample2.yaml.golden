---
kind: pipeline
name: default

platform:
  os: linux
  arch: amd64

steps:
- name: Test
  image: node:latest
  commands:
  - npm install
  - npm test

services:
- name: postgres
  image: postgres:9.6.4

...
